rest-api/src/main/java/gov/cms/qpp/conversion/api/model/Metadata.java

```diff
@@ -37,14 +39,14 @@
 	private String validationErrorLocator;
 	private String rawValidationErrorLocator;
 
-	private Date createdDate;
+	private LocalDateTime createdDate;
```

Comment:
I feel that an `Instant` would be better.  That way you don't need to convert from a `String` to an `Instant` to a `LocalDateTime`.  Also, a `LocalDateTime` has no timezone associated with it, so it is better suited for a relative time that will be different in each timezone.  If I am correct, an `Instant` already associates itself with UTC and represents an actual precise time.

Reply:
This can be done.

Reply:
Yeah. It might be more optimal with Instant. I agree and will change this.